catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/CswFilterDelegate.java

```diff
@@ -1353,16 +1353,13 @@ private void areLogicalOperationsSupported() {
      * @param filterCapabilities The {@link net.opengis.filter.v_1_1_0.FilterCapabilities} understood by the Csw service
      */
     private final void updateAllowedOperations(FilterCapabilities filterCapabilities) {
-        comparisonOps =
-                Collections.newSetFromMap(new ConcurrentHashMap<ComparisonOperatorType, Boolean>(new EnumMap<ComparisonOperatorType, Boolean>(
-                        ComparisonOperatorType.class)));
+        comparisonOps = Collections.newSetFromMap(new ConcurrentHashMap<>(new EnumMap<>(
+                ComparisonOperatorType.class)));
 
-        spatialOps =
-                new ConcurrentHashMap<SpatialOperatorNameType, SpatialOperatorType>(new EnumMap<SpatialOperatorNameType, SpatialOperatorType>(
-                        SpatialOperatorNameType.class));
+        spatialOps = new ConcurrentHashMap<>(new EnumMap<>(SpatialOperatorNameType.class));
         logicalOps = true;
         if (null == filterCapabilities) {
-            LOGGER.error("CSW Service doesn't support any filters");
+            LOGGER.info("CSW Service doesn't support any filters");
```

Comment:
If this is something that can only ever happen once when configuring something and not on every single request, then it should be info, otherwise debug.